src/PhpSpec/CodeGenerator/Generator/templates/method_signature.template

```diff
@@ -0,0 +1,3 @@
+
+    public function %name%(%arguments%);
```

Comment:
this template generates a method signature for an interface only, not for a class. So it may need a better name.